src/Twitter.php

```diff
@@ -39,20 +39,34 @@
 
 class Twitter
 {
-    use FormattingHelpers,
-        AccountTrait,
-        BlockTrait,
-        DirectMessageTrait,
-        FavoriteTrait,
-        FriendshipTrait,
-        GeoTrait,
-        HelpTrait,
-        ListTrait,
-        MediaTrait,
-        SearchTrait,
-        StatusTrait,
-        TrendTrait,
-        UserTrait,
+    use FormattingHelpers;
+    use
+        AccountTrait;
```

Comment:
i think the `use` should be on the same line as the classname?

Reply:
It should indeed. Will check why this formatting is happening and fix it

Reply:
Fixed